packages/web-config-server/src/export/excelFormatters/formatMatrixDataForExcel.js

```diff
@@ -90,7 +97,11 @@ export const formatMatrixDataForExcel = (
     });
   } else {
     // This table has no row categories, just one set of rows
-    formattedData.push(...rows.map(formatRowData));
+    formattedData.push(
+      ...rows
+        .filter(row => (categoryPresentationOptions?.showNestedRows ? !row.category : true))
```

Comment:
After investigation, found that `rowCategories` config is from [tableOfValuesForOrgUnits.js line 48](https://github.com/beyondessential/tupaia/blob/857481c74d59f537026e59aa8927269a86ee7b35/packages/web-config-server/src/apiV1/dataBuilders/generic/table/tableOfDataValues/tableOfValuesForOrgUnits.js#L48), which is no longer used in code, now we put categories config in `rows` object. 

I guess we used to have categories as a parameter sent back from backend. Do you suggest we delete these code?

Reply:
Great! Yes please delete that code, and add corresponding regression testing notes. We will need to test both exported matrices that have row categories, and some that do not.

Reply:
In that case, should we exclude category rows only when `showNestedRows` is set? Or in every case? I would expect the latter

Reply:
I was having the same thought and I hesitate to do the latter one, because we have rows average data in a category in a few matrix tables. 

Do you think they should be removed?

Reply:
Could you attach an example of an exported spreadsheet?

Reply:
Found it but seems like they are not well organised.. Happy to exclude them 👍 

[export-explore-spar-(pacific-island-countries) (1).xlsx](https://github.com/beyondessential/tupaia/files/6269518/export-explore-spar-.pacific-island-countries.1.xlsx)

Reply:
Now that I think of it, instead of excluding the categories we should just properly format them. It seems that the functionality that existed in [the original code](https://github.com/beyondessential/tupaia/blob/dev/packages/web-config-server/src/export/excelFormatters/formatMatrixDataForExcel.js#L80) took care of it, but broke at some point.

Could you please a new `Bug` card for this? We can then ignore this issue for now - it's not a bug you introduced.

Reply:
With that in mind, let's revert the changes in this file

Reply:
I will make a bug card for this. If revert we will show color in excel file like [comment](https://github.com/beyondessential/tupaia/pull/2273#issuecomment-811655332), or is it fine for now?

Reply:
It is fine for now, since it's an existing bug. Shouldn't be too hard to solve, but it's better to tackle one issue at a time 😄